src/client/utilities.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = {
+
+  /**

```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Missing JSDoc parameter type for 'givenObject'.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2159922278/issues/source?bid=3148782&fileBranchId=3159641#l5)